src/nbody/nbody.cpp

```diff
@@ -70,9 +72,10 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
   if (integrator == "none") dt_reb = Big<Real>();
   params.Add("dt_reb", dt_reb);
 
-  // Unit system for gravity
-  const Real GM = pin->GetReal("gravity", "gm");
-  params.Add("GM", GM);
+  // Total mass of gravitating particles
+  const Real gm = constants.GetGCode() * pin->GetReal("gravity", "mass_tot") *
```

Comment:
I would think you would just multiply by `GetMCode`, not `GetMsolarCode`. `G` and `M` are set in such a way that in code units `GM=1` (for the `ppd` system).

Reply:
Or maybe user inputs `GM` in code units and the mass unit, and then `G` is set from those two.

Reply:
`G` is just `G`, so I strongly prefer respecting the value of that constant provided by the underlying scale (either `G=1` for `scalefree` or `G=6.67ewhatever` for `cgs` rather than letting the user override this in a maybe unexpected way (same way this PR no longer allows users to provide custom physical constants). With that requirement the only path forward I see is to have the user provide the total gravitating mass here, and let Artemis add the `G`. 

We can allow any units for `mass_tot` here, `Msolar` seemed like a reasonable choice since that's more likely to lead to round numbers in input files, but if you think there's a better default I don't have a strong opinion, except that I think we don't want the default to be in code units because code units aren't really for the user to interact directly with in general; we don't want people to have to juggle length/mass/time conversions when they're writing input failes. Also, for the `scalefree` backend `constants.GetMSolar() = 1` so in that case the user is just inputting trivially in code units (like current input decks). There isn't a `constants.GetMCode()` method because that would just return `1` (really it would be `units.GetMassCodeToCode()` which we dont provide because it's `=1`).

Reply:
As discussed this afternoon in person, let's remove the `orbit` units with custom radius/gravitating mass, keep `ppd` units, and keep the `M` input parameter for specifiying the `GM` normalization since this mass is independent of the unit systems and must be specified somewhere in `nbody` gravity.